Packages/com.unity.inputsystem/InputSystem/Editor/ControlPicker/InputControlPickerDropdown.cs

```diff
@@ -197,6 +197,10 @@ private void AddDeviceTreeItemRecursive(InputControlLayout layout, AdvancedDropd
 
                     AddCharacterKeyBindingsTo(byCharacterGroup, keyboard);
                     AddPhysicalKeyBindingsTo(byLocationGroup, keyboard, searchable);
+
+                    // AnyKey won't appear in either group. Add it explicitly.
+                    AddControlItem(deviceItem, null,
+                        layout.FindControl(new InternedString("anyKey")).Value, layout.name, null, searchable);
```

Comment:
Minor, unrelated fix. When I switched the control picker to a special path for keyboards, I didn't realize that now `anyKey` got culled from the list. This change adds the control back to the list.